.github/workflows/install-clang-tools.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Copyright 2021 The Verible Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+sudo apt-get install clang-format
```

Comment:
Unless this is used somewhere else, I would not recommend adding it as a separate file. Just put it in the workflow. In the current state, this script is platform specific, so not really very useful outside of that specific CI job. If/when it is modified for supporting installation on different environments, then it'll make sense to have it be a file.

Reply:
Sounds good. Done.